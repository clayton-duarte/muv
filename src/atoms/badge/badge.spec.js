import React from 'react';

import Badge from '.';

it('should render a badge', () => {
  const wrapper = mount(
    <Badge />,
  );
  expect(wrapper).toMatchSnapshot();
});
