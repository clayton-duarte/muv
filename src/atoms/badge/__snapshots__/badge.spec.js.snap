// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a badge 1`] = `<Badge />`;
